packages/angular_devkit/build_angular/src/builders/application/build-action.ts

```diff
@@ -54,7 +55,7 @@ export async function* runEsBuildBuildAction(
   } = options;
 
   if (deleteOutputPath && writeToFileSystem) {
-    await deleteOutputDir(workspaceRoot, outputPath, ['browser', 'server']);
+    await deleteOutputDir(workspaceRoot, outputOptions);
```

Comment:
```suggestion
    await deleteOutputDir(workspaceRoot, outputOptions.base, [outputOptions.browser, outputOptions.server]);
```
This would remove the need to change the `deleteOutputDir` file and remove the circular dependencies.

Reply:
This is now done internally in the `deleteOutputDir` method.

Reply:
Updated